docs/_sass/_main.scss

```diff
@@ -577,6 +577,10 @@ button {
 .get-help {
     flex-wrap: wrap;
     margin-top: auto;
+
+    h3 {
+        padding-bottom: 8px;
```

Comment:
We don't need this now that we switched to `h2` right?

Reply:
Oh that's correct, I removed it